src/Form/test-cases/www_reddit_com_signup.html

```diff
@@ -97,15 +97,15 @@
                 <input type="password" data-description="dummyPassword" data-empty="true" autofill-information="overall type: PASSWORD
 server type: PASSWORD
 heuristic type: UNKNOWN_TYPE
-label: 
-parseable name: 
+label:
+parseable name:
 section: fakeemailremembered_0_15
 field signature: 475301429
 form signature: 9486430941283870347
 form signature in host form: 9486430941283870347
 field frame token: EE88768FD088DBFD37B69597958CA8DC
 form renderer id: 3
-field renderer id: 20" data-ddg-testresultelementid="582940" data-manual-scoring="password">
+field renderer id: 20" data-ddg-testresultelementid="582940">
```

Comment:
❓ Is the manual scoring removed as this input is not longer matched?

Reply:
Correct. We don't want to match this as it's a hidden dummy field they use internally, it's not shown to users and we explicitly try to skip it.